src/resolvers/Mutation/createDirectChat.ts

```diff
@@ -48,7 +48,7 @@ export const createDirectChat: MutationResolvers["createDirectChat"] = async (
       _id: args.data.organizationId,
     }).lean();
 
-    await cacheOrganizations([organization!]);
+    if (organization) await cacheOrganizations([organization]);
```

Comment:
You have added this `if (organization)` check in all of the caching statements to remove the use of the `!` operator. Still, the same is unnecessary as it is guaranteed to exist, and thus the use of the `!` operator. This change adds unnecessary verbosity to the code.
We should revert this change or find a more elegant way to deal with the same. What do you think @kb-0311?

Reply:
Ok got it. I think I need to find a better way to remove the `!` operator

Reply:
Hey, for this can we do something like this?: we can check if we are getting `id` from the `args` and if not we can throw an error. So this will make sure that that the ids we are passing to the `findOrganizationsInCache` function will never be null.